libraries/rig-package/README.md

```diff
@@ -0,0 +1,199 @@
+# @rushstack/rig-package
+
+The **config/rig.json** file is a standard that Node.js build tools can support, in order to eliminate
```

Comment:
Should we call this a "standard" yet?

Reply:
We don't have to. But I did want to communicate that non-Rush-Stack projects are invited to implement it. And the NPM package is lightweight specifically to make that easier.